__tests__/app.test.js

```diff
@@ -114,3 +114,61 @@ describe("ALL categories of undefined path", () => {
       });
   });
 });
+
+describe("GET api/reviews/:review_id/comments,", () => {
+  test("Responds with the comments of a specific review ID", () => {
+    const ID = 3;
+    return request(app)
+      .get(`/api/reviews/${ID}/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+
+        expect(comments).toEqual([
```

Comment:
I like the thinking, but consider what may happen if we at any point modify our database to test adding a new column for example. We would need to go back and change this test.
A more appropriate test would be to iterate over the responses and expect each result to match a pattern. (without using toEqual)